docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -528,6 +528,42 @@ file suffix.
 
     :return bytes: Serialized PKCS12.
 
+PKCS7
+~~~~~
+
+.. currentmodule:: cryptography.hazmat.primitives.serialization.pkcs7
+
+PKCS7 is a format described in :rfc:`2315`, among other specifications. It can
+contain certificates, CRLs, and much more. PKCS7 files commonly have a ``p7b``,
+``p7m``, or ``p7s`` file suffix but other suffixes are also seen in the wild.
+
+.. note::
+
+    ``cryptography`` only supports parsing certificates from PKCS7 files at
+    this time.
+
+.. function:: pem_load_certificates(data)
```

Comment:
Don't love these names.

Reply:
(If we were to go for max consistency it'd be `load_pem_certificates` but that seems too confusing compared to `x509.load_pem_certificate`)

Reply:
Revisiting this, we currently have `load_pem_x509_certificate` as our x509 loader function under cryptography.x509. Should we name this `load_pem_pkcs7_certificates` instead? I like that better than the current name.